README.md

```diff
@@ -393,6 +393,25 @@ Calls [`client.close(callback)`](#close) on all the clients.
 
 Calls [`client.destroy(err, callback)`](#destroy) on all the clients.
 
+<a name='errors'></a>
+### `undici.errors`
+
+Undici exposes a variety of error objects that you can use to enhance your error handling.
+You can find all the error objects inside the `errors` key.
+
+```js
+const { errors } = require('undici')
+```
+
+| Error                  | Error Codes              | Description                                            |
+| ---------------------- |--------------------------|--------------------------------------------------------|
+| `ConfigurationError`   |  `UND_ERR_CONFIGURATION` | Generated when in case of a bad configuration.         |
+| `TimeoutError`         |  `UND_ERR_TIMEOUT`       | Generated when a request exceeds the `timeout` option. |
+| `RequestAbortedError`  |  `UND_ERR_ABORTED`       | Generated in case of an aborted request.               |
```

Comment:
I guess this should only be if the user aborted the event?

Server aborts I guess falls under socket error?